src/MUIDataTableHeadCell.js

```diff
@@ -5,50 +5,55 @@ import TableCell from "@material-ui/core/TableCell";
 import TableSortLabel from "@material-ui/core/TableSortLabel";
 import Tooltip from "@material-ui/core/Tooltip";
 import { withStyles } from "@material-ui/core/styles";
+import { OptionPropTypes } from "./propTypes";
+import styled from "./styled";
 
-const defaultHeadCellStyles = {
+const defaultHeadCellStyles = (theme, props) => ({
   root: {},
-  fixedHeader: {
-    position: "sticky",
-    top: "0px",
-    left: "0px",
-    zIndex: 100,
-    backgroundColor: "#FFF",
-  },
-  tooltip: {
-    cursor: "pointer",
-  },
-  mypopper: {
-    "&[data-x-out-of-boundaries]": {
-      display: "none",
+    fixedHeader: {
+      position: "sticky",
+      top: "0px",
+      left: "0px",
+      zIndex: 100,
+      backgroundColor: "#FFF",
     },
-  },
-  data: {
-    display: "inline-block",
-  },
-  sortAction: {
-    display: "inline-block",
-    verticalAlign: "top",
-    cursor: "pointer",
-    paddingLeft: "4px",
-    height: "10px",
-  },
-  sortActive: {
-    color: "rgba(0, 0, 0, 0.87)",
-  },
-  toolButton: {
-    height: "10px",
-    outline: "none",
-    cursor: "pointer",
-  },
-};
+    fixedHeaderAndToolbar: {
+      top: "64px",
```

Comment:
64 px is a `min-height` I found for the toolbar I found in my developer console. However, I can't seem to find it *anywhere* in the codebase. It works in all of my testing, but it seems we might have different min heights depending on the orientation. Is there any way to do this more consistently?

Reply:
Setting the table responsive mode to 'scroll' you'll see this approach won't work. The true answer has to be something along the lines of taking the height of the toolbar and using that to get the true value to set `top` to

![screen shot 2018-11-09 at 7 13 39 am](https://user-images.githubusercontent.com/19170080/48262043-168f9e80-e3ef-11e8-83fb-1d51358b959b.jpg)

Reply:
That's true. Do you have a good idea of how to go about this?

Reply:
Take the computed height of the toolbar and then pass that as a prop. You can use props within the styles object since I'm using a modified withStyles HOC.  With that you could do something like:

``` 
   fixedHeaderAndToolbar: {
     top: props.toolBarHeight
   }
```

Reply:
Hey @GeeWee , any update on this?